practice-one/src/components/button-icon/index.tsx

```diff
@@ -0,0 +1,18 @@
+const ButtonIcon = (props: ButtonIconProps) => {
+  const { source, alt, primary } = props
+  return (
+    <button
+      className={`cursor-pointer rounded-full p-2 hover:shadow-sm ${primary ? 'bg-red-500' : ''}`}
+    >
+      <img src={source} alt={alt} />
+    </button>
+  )
+}
+
+type ButtonIconProps = {
+  primary?: boolean
```

Comment:
why use `primary`, how about if you have many types for one component?

Reply:
Because there are just two types for this component

Reply:
should use `variant` then pass type primary or secondary...

Reply:
I fixed @huyenpham-agilityio . Please help me check. Thanks!